src/styles/blocks/card-catalog.scss

```diff
@@ -0,0 +1,16 @@
+.card-catalog {
+  margin: 0 auto;
+  padding: 0 40px 50px;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: getWidthCatalog($card-width, $card-right-margin, $card-row);
+}
+
+.card-catalog .card:not(:nth-child(-n + 4)) {
```

Comment:
```suggestion
.card-catalog__card:not(:nth-child(-n + 4)) {
```

Reply:
this selector .card-catalog__card doesn't work. 
& .card:not(:nth-child(-n + 4)) - works, I don't know why